lib/prawn/view.rb

```diff
@@ -69,6 +69,17 @@ def method_missing(m, *a, &b)
       document.send(m, *a, &b)
     end
 
+    # Checks if +document+ respond_to +method_name+.
+    #
+    # This will correctly returns if Prawn::View object respond or not to a
```

Comment:
Good catch, lets change this to something like 

```ruby
    # Checks if +Prawn::View+ respond_to +method_name+.
    #
    # Returns `true` if the Prawn::View object responds to the method, otherwise
    # returns `false`
```

Does that look good to everyone?